css/style.css

```diff
@@ -0,0 +1,1085 @@
+  list-style-type: none;
+}
+
+.navlist li {
+  margin-right: 15px;
+}
+
+.navlist a {
+  text-decoration: none;
+  color: #aaa;
+}
+
+.footbottom a {
+  text-decoration: none;
+  color: rgba(167, 169, 169, 1);
+}
+
+.footnav a {
+  text-decoration: none;
+  text-transform: uppercase;
+  color: rgba(167, 169, 169, 1);
+  margin-right: 10px;
+}
+
+.flexed-desc a {
+  text-decoration: none;
+}
+
+.sec-4-big-pic-third a {
+  text-decoration: none;
+}
+
+.sec-4-big-pic-second a {
+  text-decoration: none;
+}
+
+.sec-4-big-pic-first a {
+  text-decoration: none;
+}
+
+.textfoot a {
+  font-family: "Varela Round", sans-serif;
+  font-size: 12px;
+  color: rgba(36, 36, 36, 0.9);
+}
+
+.title a {
+  font-family: "Russo One", sans-serif;
+  text-decoration: none;
+  color: #f42;
+  font-size: 28px;
+  font-weight: bolder;
+}
+
+.item a {
+  text-decoration: none;
+  color: black;
+  font-size: 16px;
+  font-family: "Varela Round", sans-serif;
+  font-weight: bold;
+}
+
+.yellow-nav a {
+  margin-left: 15px;
+  text-decoration: none;
+  color: black;
+  font-size: 15px;
+}
+
+.dealtitlediv p a {
+  text-decoration: none;
+  color: #f42;
+}
+
+.navlist a:hover {
+  color: white;
+}
+
+#whitetext {
+  color: white;
+}
+
+/* navlogom&s style */
+.navlogom-m {
+  background: linear-gradient(to bottom right, #c09, #f42 85%);
+  width: 106px;
+  height: 55px;
+  text-align: center;
+  padding-top: 25px;
+  display: none;
+  top: 0;
+  left: 0;
+}
+
+.navlogom-m img {
+  width: 80%;
+}
+
+.navlogom-s {
+  background: linear-gradient(to bottom right, #c09, #f42 85%);
+  width: 85px;
+  height: 25px;
+  padding-top: 15px;
+  padding-left: 20px;
+  display: none;
+}
+
+.navlogom-s img {
+  width: 100%;
+}
+
+/* navigation fixed styling */
+.header {
+  position: sticky;
+  top: 0;
+  width: 100%;
+  z-index: 100;
+}
+
+.navfixed {
+  height: 50px;
+  font-family: "Nova Square", cursive;
+  font-size: 17px;
+  background-color: white;
+}
+
+#sideinfologo {
+  width: 125px;
+  height: 54px;
+}
+
+.sideinfo {
+  margin-right: 25px;
+}
+
+.time {
+  margin-right: 20px;
+  font-size: 120%;
+  color: red;
+  font-weight: bold;
+}
+
+.ticket {
+  background-color: #0cc;
+  color: white;
+  width: 90px;
+  height: 30px;
+  font-size: 60%;
+  border: none;
+  border-radius: 3px;
+  font-weight: bold;
+}
+
+.ticket:hover {
+  background-color: purple;
+}
+
+#searchbtn {
+  transform: rotate(90deg);
+  color: #aaa;
+}
+
+#hamburger {
+  display: none;
+}
+
+.navli2 a:hover {
+  color: black;
+}
+
+/* yellow nav styling */
+
+.yellow-nav {
+  background-color: #ffe712;
+  height: 20px;
+  padding: 5px;
+}
+
+/* Styles for the main news section */
+
+/* styling for making the sections a grid layout */
+
+.main-news-section {
+  display: grid;
+  grid-gap: 6px;
+  grid-template-areas:
+    "main side-top "
+    "main side-bottom";
+  height: 50vh;
+  margin: 4px;
+}
+
+.main-pic {
+  /* background:  linear-gradient(rgba(196, 32, 32, 0.25)),url("./images/main-pic-big.png"); */
```

Comment:
Please remove this commented piece of code.

Reply:
Hey, Thank you for your detailed constructive review. It is really tough to identify those mistakes by ourselves. Your review helped us in making productive changes in our project. Based on your review, We have fixed the black navigation bar links icon problem, We have updated the main tnw img location and it is now visible. We also have changed the text alignment of the yellow nav link to the left. Images beside the Latest part are also updated and fixed. We reduced the margin-right of the latest deal section in the smaller screen, We have fixed the footer navigation links alignment and added the twn logo at the bottom of the footer. Hopefully, you will find our project acceptable after the changes that we made. Thanks again for your inspiring review and for giving us the motivation to finish the project accurately.

Reply:
Thank you for your kind comment :heart:

I hope you guys can keep up the good work :muscle: 

Keep going and never stop, you will reach the :top: 

Have a very nice day :sun_with_face: